pkg/visor/visor.go

```diff
@@ -6,6 +6,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/SkycoinProject/skywire-mainnet/internal/netutil"
```

Comment:
Could you please sort imports here and in other files?

Reply:
Sorted